integrations/roadie/manifest.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Backstage, via Roadie",
+  "version": "1.0.0",
+  "overview": "Visualize feature flag changes for software components in Backstage, via Roadie.",
+  "description": "Add a LaunchDarkly feature flag changes to related software components in Backstage, via Roadie.",
+  "author": "Roadie",
+  "supportEmail": "support@roadie.io",
+  "links": {
+    "site": "https://roadie.io",
+    "launchdarklyDocs": "https://roadie.io/backstage/plugins/launchdarkly/",
```

```suggestion
    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/roadie",
```

Once that is updated I can approve and merge.